backend/.eslintrc

```
@@ -1,17 +1,3 @@
 {
-  "extends": "airbnb-base",
-  "env": {
-    "node": true,
-    "browser": true
-  },
-  "plugins": ["import"],
-  "parserOptions": {
-    "ecmaVersion": 2018
-  },
-  "rules": {
-    "no-underscore-dangle": 0,
-    "import/imports-first": ["error", "absolute-first"],
-    "import/newline-after-import": "error",
-    "linebreak-style": 0
-  }
+  "parserOptions": { "ecmaVersion": 9 }
```

```suggestion
  "parserOptions": { "ecmaVersion": 2018 }
```
Year is what's primarily used now.